app/src/main/java/com/codingblocks/cbonlineapp/LoginActivity.kt

```diff
@@ -0,0 +1,68 @@
+package com.codingblocks.cbonlineapp
+
+import android.app.Dialog
+import android.os.Bundle
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_login.*
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.intentFor
+import org.jetbrains.anko.singleTop
+
+
+class LoginActivity : AppCompatActivity(), AnkoLogger {
+
+    private val CLIENT_ID = "9047417470"
+    private val REDIRECT_URI = "http://localhost"
```

Comment:
redirect to something like https://app.codingblocks.com and whitelist that on the server. That is more secure. 
Also in manifest add that to scheme

Reply:
@championswimmer done in the next pr